src/test/kotlin/no/nav/syfo/api/AuthenticateSpek.kt

```diff
@@ -0,0 +1,96 @@
+package no.nav.syfo.api
+
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.ContentNegotiation
+import io.ktor.http.HttpHeaders
+import io.ktor.http.HttpMethod
+import io.ktor.http.HttpStatusCode
+import io.ktor.jackson.jackson
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import io.ktor.server.testing.TestApplicationEngine
+import io.ktor.server.testing.handleRequest
+import io.ktor.util.KtorExperimentalAPI
+import no.nav.syfo.ApplicationState
+import no.nav.syfo.VaultSecrets
+import no.nav.syfo.initRouting
+import no.nav.syfo.testutil.TestDB
+import org.amshove.kluent.shouldEqual
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.util.concurrent.TimeUnit
+
+@KtorExperimentalAPI
+object AuthenticateSpek : Spek({
+
+    val database = TestDB()
+    val fakeApi = embeddedServer(Netty, 9000) {
+        install(ContentNegotiation) {
+            jackson {
+                registerKotlinModule()
+                registerModule(JavaTimeModule())
+                configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+            }
+        }
+        routing {
+            fakeJWT()
+        }
+    }.start(wait = false)
+
+    afterGroup {
+        fakeApi.stop(0L, 0L, TimeUnit.SECONDS)
+        database.stop()
+    }
+
+    describe("Authenticate basicauth") {
+        with(TestApplicationEngine()) {
+            start()
+
+            application.initRouting(ApplicationState(), database, VaultSecrets(
+                    serviceuserUsername = "username",
+                    serviceuserPassword = "password",
+                    oidcWellKnownUri = "http://localhost:9000/fake.jwt",
+                    loginserviceClientId = "clientId",
+                    syfomockUsername = "syfomock",
+                    syfomockPassword = "test"
+                    ),
+                    cluster = "dev-fss"
+
+            )
+
+            it("Validerer syfomock servicebruker") {
+                with(handleRequest(HttpMethod.Delete, "/internal/nullstillSykmeldinger/aktorId") {
+                    addHeader(HttpHeaders.Authorization, "Basic c3lmb21vY2s6dGVzdA==") //syfomock:test
+                }) {
+                    response.status() shouldEqual HttpStatusCode.OK
+                }
+            }
+
+            it("Feiler ugyldig servicebruker") {
```

Comment:
Should write the unit's test description in English

Reply:
Jeg er uenig. Denne appen kommer til å vokse og jeg mener vi taper veldig mye på å ikke skrive på det språket vi faktisk bruker i hverdagen 🇳🇴 Men vi må jo ha en enighet i teamet om dette. Kjør debatt:

Reply:
🇺🇸 ?? Du mener vel i 🇬🇧 hvis vi skal bruke engelsk 😏

Reply:
Den emojien virka ikke på fredag :(